tests/testthat/test-utilities-simulation-results.R

```diff
@@ -13,7 +13,16 @@ NUMBER_OF_EXTRA_COLUMNS <- NUMBER_OF_STATIC_COLUMNS + NUMBER_OF_COVARIATES_COLUM
 context("getOutputValues")
 
 test_that("It throws an error when no valid simulation results are provided", {
-  expect_error(getOutputValues(sim, resultsPaths))
+  expect_error(getOutputValues(individualResults, "NoPath"))

```

Comment:
This test was wrongly implemented.

Reply:
oh so it was throwing an error but because the call was BS. Lol. OK